spec/forms/developers_search_form_spec.rb

```diff
@@ -0,0 +1,72 @@
+require 'rails_helper'
+RSpec.describe DevelopersSearchForm do
+  describe '#search' do
+    before do
+      programming_language_one = create(:programming_language, name: 'PHP')
+      programming_language_two = create(:programming_language, name: 'Ruby')
+      create(:programming_language, name: 'Python')
+      language_one = create(:language, code: 'en')
+      language_two = create(:language, code: 'vn')
+      create(:developer, email: 'tester@gmail.com', programming_languages: [programming_language_one], languages: [language_one, language_two])
```

Comment:
@truongle7793 
You should remove l5, l6
and change l10, l11
ex:
```
create(:developer, email: 'tester@gmail.com', programming_languages: [create(:programming_language, name: 'PHP')], languages: [language_one, language_two])
```

Reply:
@thienhv 
Thanks
I understand, since they are only use once.
I have made the change on this.

Reply:
@thienhv 
I have made change according to review, the test's description is much better now
Thanks